demo_scripts/AKSDeployment/README.md

```diff
@@ -99,15 +99,16 @@ kubectl apply -f azure-vote-start.yml
 
 
 Store the public IP Address as an environment variable with the following command:
-'export IP_ADDRESS=$(kubectl get service azure-vote-front --output jsonpath='{.status.loadBalancer.ingress[0].ip}')'
+'for run in {1..3}; do export IP_ADDRESS=$(kubectl get service azure-vote-front --output jsonpath='{.status.loadBalancer.ingress[0].ip}'); sleep 5; done'
 
 ```
-export IP_ADDRESS=$(kubectl get service azure-vote-front --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
+for run in {1..3}; do export IP_ADDRESS=$(kubectl get service azure-vote-front --output jsonpath='{.status.loadBalancer.ingress[0].ip}'); sleep 5; done
 ```
 
 Validate IP Address by running:
 'echo $IP_ADDRESS'
-After running this command you should be able to see your application running! 
+After running this command you should be able to see your application running!
+If the variable is empty then the command ran before the IP Address became available. Press b and run the above command manually. 
```

Comment:
"press b and run the above command manually" doesn't work in a written documentation.  

Might be good to think about how to handle this in interactive context. We can be much smarter than this though since we will have the expected results in the final document. This can compare the actual results to the expected results when running in interactive mode and offer to re-run failed commands.

Reply:
I agree that there are much smarter ways to do this. The goal of this was to just get a rough unpolished version our early that we can get feedback on. I removed the comment and added a loop which should hopefully solve the issue